assignments/prototypes.js

```diff
@@ -32,16 +32,51 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+
 /*
   * Inheritance chain: Humanoid -> CharacterStats -> GameObject
   * Instances of Humanoid should have all of the same properties as CharacterStats and GameObject.
   * Instances of CharacterStats should have all of the same properties as GameObject.
 */
+function GameObject(attributes) {
+  // Properties
+  this.createdAt = attributes.createdAt;
+  this.dimensions = attributes.dimensions;
+}
+// Parent Methods
+GameObject.prototype.destroy = function() {
+  return 'Object was removed from the game.'
```

Comment:
Hey, Brandon! Glad we figured out the mysterious undefined logs. Here on line 48. 'Object was removed from the game'
I think this documentation link might help you see what is missing there. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals